lib/console/devices/devices.ex

```diff
@@ -70,19 +70,32 @@ defmodule Console.Devices do
     Repo.all(query)
   end
 
+  def get_device_for_hotspot_address(address) do
+    Device
+      |> where([d], d.hotspot_address == ^address)
+      |> Repo.one()
+  end
+
   def fetch_assoc(%Device{} = device, assoc \\ [:events, :organization, :channels]) do
     Repo.preload(device, assoc)
   end
 
   def create_device(attrs \\ %{}, %Organization{} = organization) do
     count = get_organization_device_count(organization)
     cond do
-      count > 9999 ->
+      organization.name !== "Discovery Mode (Helium)" and count > 9999 ->
         {:error, :forbidden, "Device limit for organization reached"}
       true ->
-        %Device{}
-        |> Device.create_changeset(attrs)
-        |> Repo.insert()
+        cond do
+          attrs["hotspot_address"] != nil ->
```

Comment:
in the regular create_device controller action, we need to drop the hotspot_address attr because hackers could manually add it in and hijack through certain api tools

Reply:
same in the v1 device controller

Reply:
do we need to in `v1` since the params specified and thus it wouldn't match to the `create` function and fail?

Reply:
I'll test though :-)

Reply:
you are correct, v1 does not need!

Reply:
addressed 👍 `v1` not needed per se but I added just to be safe.